src/data/showcases.js

```diff
@@ -1202,6 +1202,15 @@ export const Showcases = [
     source: null,
     tags: ["governance", "analytics"],
   },
+  {
+    title: "Multisig Platform",
+    description:
+      "Secure your treasury and participant in governance, as a team with multi-signature.",
+    preview: require("./showcase/mesh-multisig-platform.jpg"),
+    website: "https://multisig.meshjs.dev/",
```

Comment:
```suggestion
    website: "https://multisig.meshjs.dev/features",
```
Using this page instead would immediately give new users & even _prospective_ users a digest of available features... _as well as_ the `Connect:` button which is the only feature on the former page.